src/glmtools.jl

```diff
@@ -384,6 +384,9 @@ the signed square root of this value
 sign(y - μ) * sqrt(devresid(D, y, μ))
 ```
 
+Note that the residuals can also be obtained by simply doing `residuals(model)` --- this convenience function would return the same vector of residuals that we'd get from the code block above involving `devresid`, if `μ == predict(model)` and both `y` and `D` are the same as those used with the original `lm` call. 
```

Comment:
Sorry for the repeated suggestions, but this would be more appropriate in a docstring for `residuals` (to be added). Here, this would probably be enough:
```suggestion
See also: [`residuals`](@ref)
```